openmeetings-web/src/main/java/org/apache/openmeetings/web/app/Application.properties.xml

```diff
@@ -960,4 +960,8 @@ see https://openmeetings.apache.org/LanguageEditor.html for Details
 	<entry key="wizard.button.finish"><![CDATA[Finish]]></entry>
 	<entry key="zoom.FULL_FIT"><![CDATA[Full-Fit]]></entry>
 	<entry key="zoom.PAGE_WIDTH"><![CDATA[Page Width]]></entry>
+	<entry key="wb.tool.dweet"><![CDATA[Dweet IOT Messaging]]></entry>
+	<entry key="wb.tool.dweet.mything"><![CDATA[MyThing]]></entry>
```

Comment:
It is not clear what `MyThing` should mean

Reply:
"MyThing "   is  the  term  used  by Dweet  for  an  object ( fridge, light )   to  be  controlled. 

To make this  clearer for the user we   could  add an  example  object in the text  field:      MyThing:   ..Light ...........